docs/pages/router/reference/troubleshooting.mdx

```diff
@@ -44,6 +44,16 @@ export function App() {
 registerRootComponent(App);
 ```
 
+You will need to also update your app's main entry point:
```

Comment:
```suggestion
Then, update your app's main entry point in **package.json**:
```

Since this follows from the context of the previous step, let's make sure to imply that and mention the file name.